crates/rust-client/src/store/account.rs

```diff
@@ -17,18 +18,20 @@ pub struct AccountRecord {
     account: Account,
     /// Status of the tracked account.
     status: AccountStatus,
+    /// Addresses that the account can be referred as.
```

Comment:
nit (feel free to disregard):

```suggestion
    /// Addresses by which this account can be referenced.
```